src/Commands/RunCommand.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: nitishkumar
+ * Date: 3/8/18
+ * Time: 9:51 AM
+ */
+
+namespace Hyn\Tenancy\Commands;
+
+use Hyn\Tenancy\Contracts\Repositories\WebsiteRepository;
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Console\Kernel;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Hyn\Tenancy\Models\Website;
+
+class RunCommand extends Command
+{
+    protected $signature = 'tenancy:run {tenantcommand}
+        {--tenant= : The tenant(s) to apply on; use {all|5,8}}
```

Comment:
In the current version of Laravel, we can use `{--tenant=*}` to allow for multiple occurences of the option. This means we can call the command like `php artisan tenancy:run --tenant=1 --tenant=2 --tenant=7`. `$this->option('tenant')` will return `[1, 2, 7]` in this case.

Personally, I think the above is cleaner, but I see a point for your solution due to its brevity. It could be discussed whether `--tenants=` (plural) would be better, though, as the command description suggests two use cases where more than one tenant is addressed.